tensorflow/core/ops/math_ops.cc

```diff
@@ -478,6 +478,14 @@ REGISTER_OP("Ceil")
 Returns element-wise smallest integer in not less than x.
 )doc");
 
+REGISTER_OP("Rint")
+    .Input("x: T")
+    .Output("y: T")
+    .Attr("T: {float, double}")
+    .Doc(R"doc(
+Returns element-wise integer closest to x.
```

Comment:
The result of the operation depends on the underlying C++ implementation and default setting.

To be certain at all the times what is our rounding, we can explicitly set the `std::fesetround(FE_TONEAREST);` when entering the C++ context and restore the previous mode if is not set to FE_TONEAREST.
```
save_rint_setting = std::fegetround();
std::fesetround(FE_TONEAREST);
std::rint(value);
std::fesetround(save_rint_setting);
```
### Default settings 
- The gnu library is set to `FE_TONEAREST` by default.
http://www.gnu.org/software/libc/manual/html_node/Rounding.html
- The cudnn library _seems_ to use a logic like TONEAREST http://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#standard-functions but is not something explicitly set.
I can just just ignore this entire issue and say in the description that does rounding 'to nearest'.